_data/cryptocurrencies.yml

```
@@ -502,6 +502,13 @@ websites:
     regions:
       - br
 
+  - name: NairaEx
+    url: https://nairaex.com/
+    img: nairaex.svg
+    tfa:
+      - totp
+    doc: https://support.nairaex.com/support/solutions/articles/43000335554
+
```

```suggestion
    regions:
      - ng

```

The website mentions 

> Nigeria's Premier Bitcoin Exchange

Is this site available outside of Nigeria?